extensions/MESA/MESA_framebuffer_flip_y.txt

```diff
@@ -59,7 +59,10 @@ Issues
 
 New Procedures and Functions
 
-    None
+    If ARB_framebuffer_no_attachments is not implemented then implement
+
+    void FramebufferParameteriMESA(enum target, enum pname, int param);
+    void GetFramebufferParameterivMESA(enum target, enum pname, int *params);
```

Comment:
About the 'New Tokens' section below...

I believe it should read as follows, plus an "dependency interaction" section is needed.


```
New Tokens

    Accepted by the <pname> argument of FramebufferParameteriMESA and
    GetFramebufferParameterivMESA:

        FRAMEBUFFER_FLIP_Y_MESA                      0x8BBB

Dependencies on OpenGL 4.3, OpenGL ES 3.1, ARB_framebuffer_no_attachments, and
any other versions and extensions that provide the entry points FramebufferParameteri
and FramebufferParameteriv

        Token FRAMEBUFFER_FLIP_Y_MESA is accepted as the <pname> argument of
        FramebufferParameteri and FramebufferParameteriv.
```